README.md

```diff
@@ -12,96 +12,96 @@ Pull requests are welcome. Please keep the list in alphabetical order.
-[Clevertech](https://clevertech.biz/)             | Dev consulting                           | Many (see job page)                            | ✗                                    | ?                       | :keyboard: [Jobs](https://hire.clevertech.biz/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Clevertech-EI_IE781853.11,21.htm)
-[Close](https://close.com/)                       | Insides sales CRM                        | Python, JavaScript, Web                        | ✓                                    | ?                       | :keyboard: [Jobs](http://jobs.close.com)<br>:door: [Glassdoor](https://www.glassdoor.com/Reviews/Close-io-Reviews-E1155591.htm)
-[Cloudbeds](https://www.cloudbeds.com/)           | Hospitality Software                     | Web, DevOps                                    | ✓                                    | ?                       | :keyboard: [Jobs](https://www.cloudbeds.com/careers)<br>:door: [Glassdoor](https://www.glassdoor.com/Overview/Working-at-Cloudbeds-EI_IE1399374.11,20.htm)
-[Cloudbees](https://www.cloudbees.com/)           | CI/CD, Jenkins, Codeship                 | Java, Web, Go, Vuejs, Rails                    | ✓                                    | ?                       | :keyboard: [Jobs](https://www.cloudbees.com/careers)<br>:door: [Glassdoor](https://www.glassdoor.ca/Overview/Working-at-CloudBees-EI_IE1106279.11,20.htm)
+[Chef](https://www.chef.io/)                      | IT automation                            | Ruby, DevOps                                   | ✓                                    | Limited remote          | :keyboard: [Jobs](https://www.chef.io/careers/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Chef-Software-EI_IE444809.11,24.htm)
+[chess.com](https://www.chess.com)                | Online chess                             | Java, mobile, etc..                            | ?                                    | Anywhere                | :keyboard: [Jobs](https://www.chess.com/jobs)
+[Clevertech](https://clevertech.biz/)             | Dev consulting                           | Many (see job page)                            | ✗                                    | Anywhere                | :keyboard: [Jobs](https://hire.clevertech.biz/)<br>:door: [Glassdoor](https://www.glassdoor.com.au/Overview/Working-at-Clevertech-EI_IE781853.11,21.htm)
+[Close](https://close.com/)                       | Insides sales CRM                        | Python, JavaScript, Web                        | ✓                                    | Mostly US or EU only    | :keyboard: [Jobs](http://jobs.close.com)<br>:door: [Glassdoor](https://www.glassdoor.com/Reviews/Close-io-Reviews-E1155591.htm)
```

Comment:
What does "mostly"  mean here?

Reply:
I got confused - engineering jobs are only US or EU, but those other remote jobs did not specify a location/timezone. Will just put US or EU only.